tests/sentry/api/endpoints/test_project_metrics_extraction_rules.py

```diff
@@ -75,6 +76,12 @@ def test_create_new_extraction_rule(self):
             assert data[0]["aggregates"] == ["count"]
             assert data[0]["unit"] == "none"
             assert set(data[0]["tags"]) == {"tag1", "tag2", "tag3"}
+            assert data[0]["createdById"] == self.user.id
+
+            deserialized_date_added = datetime.fromisoformat(response.json()[0]["dateAdded"])
+            deserialized_date_updated = datetime.fromisoformat(response.json()[0]["dateUpdated"])
+            assert deserialized_date_added == data[0]["dateAdded"]
```

Comment:
Nit: This can also be done by using `freeze_time` decorator on this test case, and making it little bit clearer what is being compared :)

Reply:
good point, changed it